agent/custom/general_navigation_action.py

```
@@ -1,238 +1,482 @@
+    th, tw = template_bgr.shape[:2]
+    if th < 4 or tw < 4:
+        return 0.0
+    ch, cw = crop_bgr.shape[:2]
+    if ch < 4 or cw < 4:
+        return 0.0
+    crop_r = cv2.resize(crop_bgr, (max(4, int(cw * MATCH_H / ch)), MATCH_H))
+    tpl_r = cv2.resize(template_bgr, (max(4, int(tw * MATCH_H / th)), MATCH_H))
+    if tpl_r.shape[1] > crop_r.shape[1]:
+        # 模板比框宽: 框等比放大到模板宽度(不拉伸模板, 保持内容比例)
+        ratio = tpl_r.shape[1] / crop_r.shape[1]
+        crop_r = cv2.resize(crop_r, (tpl_r.shape[1], max(4, int(round(crop_r.shape[0] * ratio)))))
```

Ruff (0.16.1)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)